toolkit/tools/imagecustomizerapi/disk.go

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package imagecustomizerapi
+
+import (
+	"fmt"
+	"sort"
+	"strconv"
+
+	"github.com/microsoft/CBL-Mariner/toolkit/tools/internal/sliceutils"
+)
+
+type Disk struct {
+	// The type of partition table to use (e.g. mbr, gpt)
+	PartitionTableType PartitionTableType `yaml:"PartitionTableType"`
+
+	// The virtual size of the disk.
+	MaxSize uint64 `yaml:"MaxSize"`
```

Comment:
curious, why `MaxSize` and not just `Size`?

Reply:
Not really sure. But it's the name the new-image config uses. So, we mirror it here.